### DISCUSSION TITLE
Just curious about the upcoming Mini (not plus) hardware revision

### DISCUSSION THREAD

**Discussion by jepyang** [OnionUI/Onion#1138] [General]
I assume most of the answers are “don’t know yet” but any expectations about whether Onion will work out of the box with the new batch of Minis? There’s a new display with higher resolution, I don’t know if any other hardware revision details are known yet.

have any devs managed to place an order for the new device?

**Comment by Aemiii91** [comment]
We have all received the new version, and there's no difference software wise
